x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/investigate_in_timeline_action.tsx

```diff
@@ -22,7 +22,6 @@ interface InvestigateInTimelineActionProps {
   alertIds?: string[];
```

Comment:
We need to do a proper clean up of alertIds usage. That would be great to clarify in the PR description why we don't need any more fetching the alerts and alertIds.